README.md

```diff
@@ -95,14 +95,16 @@ You can reach the maintainers of this project at:
 
 - [Slack](https://skippbox.herokuapp.com): #kompose
 
-Incubation of `kompose` into the Kubernetes project will be discussed in the [API Machinery SIG](https://github.com/kubernetes/community)
+Incubation of `kompose` into the Kubernetes project is shepherd in the [SIG-APPS](https://github.com/kubernetes/community).
+
+[@ericchiang](https://github.com/ericchiang) is acting as Champion for incubation
```

Comment:
missing period after "incubation". perhaps add link? 
`(incupation)[https://github.com/kubernetes/community/blob/master/incubator.md]`

since i had to google what "Champion" was and got linked to the incubation doc on kubernetes community